syncr_backend/node_init.py

```diff
@@ -0,0 +1,59 @@
+import os
+import shutil
+
+from syncr_backend import crypto_util
+
+
+def force_initialize_node() -> None:
+    """Initialize new node in .node directory
+    and overwrite existing .node dir"""
+
+    if os.path.exists(".node"):
```

Comment:
make .node a global variable (ie DEFAULT_NODE_DIR), and then parametrize it (ie, `def force_initialize_node(node_dir: bytes=DEFAULT_NODE_DIR):`